docker-compose.yml

```diff
@@ -0,0 +1,90 @@
+version: "3.4"
+services:
+  conference-app:
+    container_name: conference-app
+    env_file:
+      - .env
+      - envs/conference-app.env
+    command:
+      - sh
+      - -c
+      - |
+        if [ -e tmp/pids/server.pid ]; then
+          rm tmp/pids/server.pid
+        fi
+        bundle install
+        bin/rails s -p 3000 -b 0.0.0.0
+    build:
+      context: .
+    volumes:
+      - .:/app
+      - conference-app-vendor:/vendor
```

Comment:
This configuration may not work since the gems will be placed in `/usr/local/bundle` instead of the `/vendor` directory.

Reply:
Changed the path with 0d88079 !